with-webpack.md

```diff
@@ -78,7 +78,7 @@ new PurgecssPlugin({
 
 * only
 
-You can specify entrypoints to the purgecss-webpack-plugin with the option only:
+You can specify entrypoints to the purgecss-webpack-plugin with the `option` only:
```

Comment:
Wouldn't it be better if `only` was in code format instead of `option`?

Reply:
Oh, yes! My bad! I'll change that.